docs/docs/03-language-reference.md

```diff
@@ -1771,6 +1771,28 @@ bring cloud; // from cloud bring * as cloud;
 bring "cdktf" as cdktf; // from "cdktf" bring * as cdktf;
 ```
 
+To import an individual Wing file as a module, you can specify its path relative
+to the current file:
+
+```TS
+bring "./my-module.w" as myModule;
+```
+
+It's also possible to import a directory as a module. The module will contain all
+public types defined in the directory's files. If the directory has subdirectories,
+they will be available under the corresponding names.
+
+```TS
+bring "./my-module" as myModule;
+
+// from ./my-module/submodule/my-class.w
+new myModule.submodule.MyClass();
```

Comment:
I wonder if this will create a schism in how people do imports in wing. Should we only allow one way? I suspect people would gravitate towards directory imports since they're more powerful.

Reply:
I think it would be pretty annoying if I would have to create a directory for every module.

If we eventually decide to support implicit references across files in the same directory, then this would make more sense.

Reply:
Directory imports are more powerful, but sometimes you may want to use types from sibling files, in which case importing the individual file is necessary because https://github.com/winglang/wing/issues/4276 is not supported yet